game/views.py

```
@@ -25,11 +26,74 @@
+        return JsonResponse({
+            'available': False,
+            'suggestions': [],
+            'error': 'Username must not start or end with a space.',
+        })
+
+    if not _USERNAME_RE.match(username):
+        return JsonResponse({
+            'available': False,
+            'suggestions': [],
+            'error': (
+                'Username must be 3–20 characters and may only contain '
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)